packages/aria-voyager/tests/menu/navigation/keyboard/arrow-left.test.ts

```diff
@@ -12,24 +12,24 @@ describe('Close with `ArrowLeft`', () => {
   const { firstItem, fourthItem } = getItems(menu);
 
   test('open share menu', async () => {
-    expect(shareMenu.matches(':popover-open')).toBeFalsy();
```

Comment:
please keep the `matches(':popover-open')` assertions as they were. The `toBeVisible()` is not an adequate replacement, as it requires certain CSS to be present (which isn't what this library is about).